corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -174,14 +174,21 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
             this.selections = selections;
         };
 
+        this.setSelectedValues = function (selectedValues) {
+            this.selectedValues = selectedValues;
```

Comment:
This should also get cleared - the CloudCare URL is persistent throughout the session, so when the user does something like submit a form or goes to the home screen, most of its attributes get reset. Look at where `selections` is updated in this file, you likely want to update those same places.

Reply:
Nevermind - I don't think `selectedValues` should be part of the URL, will leave a more detailed comment in a minute.

Reply:
...reversing that comment, you **do** need still to clear the selected values. Even if they're not stored on the URL, they need to be cleared when an action happens that clears the URL.